wgengine/packet/icmp.go

```diff
@@ -47,7 +47,7 @@ const (
 	icmpAllHeadersLength = ipHeaderLength + icmpHeaderLength
 )
 
-func (h *ICMPHeader) Length() int {
+func (h *ICMPHeader) Len() int {
```

Comment:
ICMPHeader is still being used as a pointer receiver? I thought the others changed to be value receivers?

Reply:
Yep, sorry, I think I mis-merged something. Fixed.